server/templates/custom_dc/climate_trace/timeline_examples.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "header": "CO2 in Manufacturing",
+    "preposition": "for",
+    "examples": [
+      {
+        "text": "India, China, US",
+        "url": "#place=country%2FUSA%2Ccountry%2FIND%2Ccountry%2FCHN&statsVar=Annual_Emissions_CarbonDioxide_Manufacturing"
+      }
+    ]
+  },
+  {
+    "header": "CO2 in Steel Manufacturing",
+    "preposition": "for",
+    "examples": [
+      {
+        "text": "Japan, China, US, Germany",
+        "url": "#place=country%2FDEU%2Ccountry%2FGBR%2Ccountry%2FJPN%2Ccountry%2FCHN&statsVar=Annual_Emissions_CarbonDioxide_SteelManufacturing&chart=%7B%22amount%22%3A%7B%22pc%22%3Atrue%7D%7D"
+      }
+    ]
+  },
+  {
+    "header": "CO2 (per capita) for Road Transportation",
```

Comment:
put per capita at the end of the SV (here and below)

Reply:
Done